### ISSUE TITLE
[Parole page] Improve keyboard navigation on StateDistrictMap

### ISSUE THREAD

**Issue by macfarlandian** [Recidiviz/public-dashboard#18]
By default it appears that the states are keyboard-navigable and the markers are not. We would like it to be the reverse. Hover states (including tooltip if applicable) should be activated upon keyboard focus on a marker.
